Anchor.toml

```diff
@@ -6,19 +6,19 @@ skip-lint = false
 
 [programs.localnet]
 multicall_handler = "6kqWTz3A3ZYMV2FMU24ke8rHzT82SaBz7GkBKTd7Z9BH"
-svm_spoke = "DnLjPzpMCW2CF99URhGF3jDYnVRcCJMjUWsbPb4xVoBn"
+svm_spoke = "YVMQN27RnCNt23NRxzJPumXRd8iovEfKtzkqyMc5vDt"
 test = "8tsEfDSiE4WUMf97oyyyasLAvWwjeRZb2GByh4w7HckA"
 
 [programs.devnet]
 multicall_handler = "6kqWTz3A3ZYMV2FMU24ke8rHzT82SaBz7GkBKTd7Z9BH"
-svm_spoke = "DnLjPzpMCW2CF99URhGF3jDYnVRcCJMjUWsbPb4xVoBn"
+svm_spoke = "YVMQN27RnCNt23NRxzJPumXRd8iovEfKtzkqyMc5vDt"
 
 [registry]
 url = "https://api.apr.dev"
 
 [provider]
-cluster = "localnet"
-wallet = "test/svm/keys/localnet-wallet.json"
```

Comment:
this would break forking tests where we have overrided CCTP state with this key as permissioned actor

Reply:
yes, was not planning on committing when the PR is ready